packages/health-gorilla-core/package.json

```diff
@@ -27,7 +16,7 @@
   },
   "main": "dist/cjs/index.cjs",
   "module": "dist/esm/index.mjs",
-  "types": "dist/esm/index.d.ts",
+  "types": "dist/cjs/index.d.ts",
```

Comment:
Same as `module` comment - although in this case they're probably the exact same content.